c10/util/Exception.cpp

```diff
@@ -127,6 +127,17 @@ WarningHandler* get_warning_handler() noexcept(true) {
   return ThreadWarningHandler::get_handler();
 }
 
+bool warn_only_once = true;
+
+void set_warn_once(bool warn_once) noexcept(true) {
+    bool ret = warn_only_once;
```

Comment:
What's `ret` for?

Reply:
leftover from a version that returned the previous value. I will remove in the next iteration.

Reply:
done